scripts/devserver

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ./node_modules/.bin/babel-node
```

Comment:
How much do you care about Javascript scripts having a `.js` file extension? Makes git diffs nice and colorful, makes it easy to see at a glance what language the scripts are in (we could also rename the bash ones to `.sh`). Thoughts?